### PR TITLE
EBeam_SergeyNomoev.gds

### PR DESCRIPTION
**Pull request by serganom**
do you see my file?

### CODE REVIEW

**Comment by lukasc-ubc** [comment]
Error: Cell bounding box / extent (615.15, 416.27) is larger than the maximum size of 605.0 X 410.0 microns

File: EBeam_SergeyNomoev_v1.gds
Comment (omidesml) [inline EBeam_SergeyNomoev_v1.gds:1]: This is pushed to the main and not to the "submission" directory.

Comment (serganom) [reply EBeam_SergeyNomoev_v1.gds:1]: but i added another file without v1, called: EBeam_SergeyNomoev.gds

Comment (serganom) [reply EBeam_SergeyNomoev_v1.gds:1]: And cell bounding box size 605.0 X 410.0 microns

Comment (serganom) [reply EBeam_SergeyNomoev_v1.gds:1]: I left only the new version, is everything ok now?

**Pull request merged**